symfony/framework-bundle/3.3/bootstrap.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require __DIR__.'/vendor/autoload.php';
+
+if (!isset($_SERVER['APP_ENV'])) {
+    $_SERVER['APP_ENV'] = isset($_ENV['APP_ENV']) ? $_ENV['APP_ENV'] : null;
+}
+
+if ('prod' !== $_SERVER['APP_ENV']) {
```

Comment:
There's an edge-case here. With no "real" env vars set, if I run

```php
php bin/console --env=prod some:command
```

None of the `.env `files (including `.env.prod.local`) are loaded. The intention of this line I think is "to avoid loading the `.env` files if real env vars are set". But passing `--env=prod` isn't a real env var. It's annoying, but should we cover that case - *yes* load the `.env` files if `--env=prod` is passed via the command line?

Reply:
works for me, that's how [it used to work actually](https://github.com/symfony/recipes/pull/481/files#diff-2af50efd729ff8e61dcbd936cf2b114b).
updated

Reply:
It almost works now. When I run `php bin/console --env=prod some:command`, it DOES load `.env`, but it does not load `.env.prod.local` because it thinks (at that moment) that the `$env` is still `dev`.

Reply:
Fixed